src/drc/api/views/bestandsdeel.py

```diff
@@ -17,8 +20,37 @@
         summary=_("Upload een bestandsdeel."),
     ),
 )
+@extend_schema(
+    responses={
+        200: inline_serializer(
+            name="CustomResponseForWriteOnlyFields",
+            fields={
+                "url": serializers.HyperlinkedIdentityField(
+                    view_name="bestandsdeel_update"
+                ),
+                "lock": serializers.CharField(
+                    help_text=BestandsDeelSerializer._declared_fields["lock"].help_text,
+                ),
+                "omvang": serializers.IntegerField(
+                    help_text=BestandsDeel.omvang.field.help_text, required=False
+                ),
+                "inhoud": serializers.FileField(
```

Comment:
In de response zou dit een URL terug moeten geven (de request verwacht een bestand)

Reply:
momenteel word dit terug gegeven:



string <uri> (inhoud)De (binaire) bestandsinhoud van dit specifieke bestandsdeel.
--